tests/Filter/FilterTest.php

```diff
@@ -102,11 +103,27 @@ public function testLabel(): void
         $this->assertSame('foo', $filter->getLabel());
     }
 
+    public function testExceptionOnNonDefinedFilterName(): void
+    {
+        $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage(sprintf(
+            'Seems like you didn\'t call `initialize()` on the filter `%s`. Did you create it through `%s::create()`?',
+            FooFilter::class,
+            FilterFactory::class
+        ));
```

Comment:
These lines should be moved right before the line that trigger the exception. This remark might apply below too.

Reply:
Fixed